spampd.pl

```diff
@@ -898,6 +927,56 @@ sub setup_logging {
 
 ##################   SERVER METHODS   ######################
 
+sub audit {
+  my ($self, $msglines) = @_;
+  my $prop = $self->{spampd};
+  my $status;
+  # Audit the message
+  if ($prop->{sa_client}) {
+    $status = $self->{assassinc}->process(\$msglines);
```

Comment:
This doesn't need any disposal/finish() ?

Reply:
assassinc returns hashmap, i am not sure if that is needed to dispose explicitly,